more/free-programming-cheatsheets.md

```diff
@@ -373,6 +373,8 @@
 #### Django
 
 * [All in one Cheatsheet of Django](https://cheatography.com/ogr/cheat-sheets/django/pdf/) - Olivier R. (PDF)
+* [The Definitive Guide To Django](https://doc.lagout.org/programmation/Django/The%20Definitive%20Guide%20To%20Django%20-%20Holovaty%2C%20Kaplan-Moss%20-%20Apress%20%282009%29.pdf) - Adrian Holovaty
+and Jacob Kaplan-Moss. (PDF)
```

Comment:
This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.